io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/writer/XLSInstanceWriter.java

```diff
@@ -73,86 +74,118 @@ protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 		boolean solveNestedProperties = getParameter(
 				InstanceTableIOConstants.SOLVE_NESTED_PROPERTIES).as(Boolean.class, false);
 
+		boolean emptyFeatureTypes = getParameter(InstanceTableIOConstants.EXPORT_EMPTY_FEATURETYPES)
+				.as(Boolean.class, false);
+
 		// get the parameter to get the type definition
 		String exportType = getParameter(InstanceTableIOConstants.EXPORT_TYPE).as(String.class);
-		QName selectedTypeName = null;
+		ArrayList<QName> selectedFeatureTypes = new ArrayList<>();
+
+		String[] array = exportType.split(",");
```

Comment:
The definition and documentation of the parameter should be adapted to the new behavior:
https://github.com/halestudio/hale/blob/5d335696fd7b8f7e2114d3423ea5d2cacf2445f2/io/plugins/eu.esdihumboldt.hale.io.xls/plugin.xml#L94